deals/deals.go

```diff
@@ -124,44 +130,58 @@ func (m *Module) Store(ctx context.Context, waddr string, data io.Reader, dcfgs
 			Success:     true,
 		}
 	}
-	return dataCid, res, nil
+	return res, nil
+}
+
+// Fetch fetches deal data to the underlying blockstore of the Filecoin client.
+// This API is meant for clients that use external implementations of blockstores with
+// their own API, e.g: IPFS.
+func (m *Module) Fetch(ctx context.Context, waddr string, cid cid.Cid) error {
+	return m.retrieve(ctx, waddr, cid, nil)
```

Comment:
Just to be more explicit about what our current implementation is actually doing (for my own understanding)... This causes a retrieval deal to be executed and adds the resulting data to the Lotus node's IPFS node?

Reply:
Correct.